tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -431,10 +440,94 @@ def test_common_workload(self):
                                            driver, (workload, validator),
                                            num_workers=self.CLUSTER_NODES - 1,
                                            topology="ensemble")
-        benchmark.start()
-        benchmark_time_min = benchmark.benchmark_time() + 5
-        benchmark.wait(timeout_sec=benchmark_time_min * 60)
-        benchmark.check_succeed()
+        # Latency spikes detection and retry
+        max_retries = 2
+        try_count = 0
+
+        while try_count < max_retries:
+            try_count += 1
+            self.logger.info(
+                f"Starting benchmark attempt {try_count}/{max_retries}.")
+            # Run the benchmark test
+
+            benchmark.start()
+            benchmark_time_min = benchmark.benchmark_time() + 5
+            benchmark.wait(timeout_sec=benchmark_time_min * 60)
+
+            #benchmark.check_succeed()
+            # check if omb gave errors, but don't process metrics
+            #benchmark.check_succeed(validate_metrics=True, )
+            is_valid, validation_results = benchmark.check_succeed(
+                raise_exceptions=False)
+
+            if is_valid:
+                self.logger.info(
+                    "\n\nBenchmark passed without significant latency issues.\n"
+                )
+                for result in validation_results:
+                    self.logger.info(f"Result is: {result}")
+                break
+
+            else:
+                self.logger.info(
+                    f"\nBenchmark test attempt {try_count} failed.\n")
+                for result in validation_results:
+                    self.logger.info(f"Result is: {result}")
+                self.logger.info(
+                    "Checking test results for possible latency spikes.")
+
+                #latency_metrics = benchmark.check_succeed(validate_metrics=False, return_latency_metrics=True, raise_exceptions=False)
+
+                latency_metrics = {
+                    key: benchmark.metrics[key]
+                    for key in self.LATENCY_SERIES_AND_MAX
+                    if key in benchmark.metrics
+                }
+
+                if latency_metrics:  # Ensure latency_metrics is not None or empty
+                    self.logger.info("\nLatency metrics for spikes detection:")
+                    self.logger.info(
+                        "\n========================================")
+                    for key, value in latency_metrics.items():
+                        # Assuming value is a list of latency values for each series
+                        series_values = ", ".join(str(v) for v in value)
+                        self.logger.info(f"    {key}: [{series_values}]")
+                    self.logger.info(
+                        "\n========================================")
+                else:
+                    self.logger.info("No latency metrics returned.")
```

Comment:
Why would this happen (be empty)? Should it be treated as a bug/failure instead?